test/CheckerTest.t.sol

```diff
@@ -21,10 +21,10 @@ contract CheckerTest is BaseFixture {
 
         delayModule.executeNextTx(EURE, 0, payload, Enum.Operation.Call);
 
-        (bool canExec, bytes memory execPayload) = roboModule.checker();
+        (bool canExec,) = roboModule.checkUpkeep("");
 
         assertTrue(canExec, "CanExec: not executable");
-        assertEq(bytes4(execPayload), ADJUST_POOL_SELECTOR, "Selector: not adjust pool (0xba2f0056)");
+        // assertEq(bytes4(execPayload), ADJUST_POOL_SELECTOR, "Selector: not adjust pool (0xba2f0056)");
```

Comment:
why is it not possible to extract the selector here anymore?

Reply:
in `ClosePoolTest.t.sol`L29 the pool action is confirmed, would that be possible here too?

Reply:
the selector is not part anymore of the `execPayload`. now we only extract either the action type and the amount

Reply:
well that is what i mean, if we are getting rid of the selector assert, shouldnt we replace it with a pool action type assert instead?

Reply:
misunderstood your first comment, that makes sense

pushed in b202b371dc989d2d596642c96d605484a6e0c1b1

Reply:
awesome